### ISSUE TITLE
[Feature]: Resolve Create 'Veridium' name conflict

### ISSUE THREAD

**Issue by Zepalesque** [Zepalesque/Aether-Redux-Issue-Tracker#132]
Most likely solution will be to add two optional resource packs built-in to the mod, one renaming this mod's veridium, one renaming create's veridium
users will be able to choose which they want to rename (if either, alternatively both)

Potential Redux Veridium renames:
- Nyridium (based on original concept name)

Potential Create Veridium renames:
- Azurite
- Chloridium

**Comment by VoidLeech** [comment]
Azurite is probably not a good alternative when renaming Create's Veridium, as Create already has Asurine for a blue-ish stone type

**Comment by Zepalesque** [comment]
yeah I actually have since making the list settled on Chloridium lol
the idea of using something similar to the word 'Chlorophyll' popped into my head and very quickly after became my favorite of them lol

**Comment by Qwerty97475** [comment]
Potentially, you could rename Create's Veridium to Malachite, which is the name of a very similar looking green rock containing copper irl?

![Image](https://github.com/user-attachments/assets/d628ab74-e2d7-449c-be98-81eb258ad8eb)
